packages/react/src/wallet/infrastructure/SignerFactory.ts

```diff
@@ -30,7 +30,7 @@ export class SignerFactory implements ISignerFactory {
     chainType,
   }: CreateSignerConfig): PromiseResult<RequiredSigner, WalletConnectionError> {
     const chainId = chainType ? this.chains[chainType].chainId : undefined;
-    const signer = await this.bindings.getSigner({ chainId });
+    const signer = await this.bindings.getSigner({});
```

Comment:
can you please clarify what this change achieve?

Reply:
I don't know if it's safe to make this assumption on all potential bindings. You mentioned an error... which probably is the cause of the issue... do you have more details on the error? Can the wagmi bindings do a best effort approach and return the desired signer or fallback to chain-less Id if not available?